.github/workflows/ci.yml

```diff
@@ -30,6 +32,15 @@ jobs:
         with:
           java-version: 17
       - run: sbt "test; scripted sbt-scalafix/*"
+  jdk21:
+    name: JDK21 tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: olafurpg/setup-scala@v14
+        with:
+          java-version: 21
```

Comment:
this was picking `graalvm@21.1.0`, which is not JDK21 - following up in https://github.com/scalacenter/sbt-scalafix/pull/396